examples/data-solutions/data-platform-foundations/02-load.tf

```diff
@@ -83,13 +83,13 @@ module "lod-prj" {
 }
 
 module "lod-vpc" {
-  count      = var.network_config.network != null ? 0 : 1
+  count      = var.network_config.network_self_link != null ? 0 : 1
   source     = "../../../modules/net-vpc"
   project_id = module.lod-prj.project_id
   name       = "${local.prefix_lod}-vpc"
```

Comment:
If `project_create` is null or false, project names should hardcoded and use prefix, as all other examples in Fabric do. It should also use the `project_ids` (with trailing `s`) variable as a flag instead of `project_create`, so project creation depends on a single variable, not two interdependent ones.

Reply:
Fixing this in PR #467 .